identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -342,7 +351,9 @@ enter.your.username=Enter your username
 error.enter.code=Please enter the code!
 choose.other.option=Choose a different option
 resend.code.success=A new OTP is sent to your mobile.
+resend.code.success.email=A new OTP is sent to your email.
 enter.code.sent.smsotp=Enter the code sent to your mobile phone
+enter.code.sent.emailotp=Enter the code sent to your email
```

Comment:
Is using the term `code` intentional? seems we have used the term `OTP` in all other places.